src/accapi/client.py

```diff
@@ -51,9 +52,11 @@ def __init__(
         self._onBroadcastingEvent = Observable()
 
         # Session properties
+        self._udpBufferSize = 1024
```

Comment:
Buffer has been implemented in a way that follow the original concepts of this repository in #4.